t/obs/openSUSE:Leap:16.0:ToTest/print_openqa.before

```diff
@@ -83,9 +113,14 @@
  CHECKSUM_ISO=$(cut -b-64 /var/lib/openqa/factory/other/Leap-16.0-offline-installer-ppc64le-Build1.3.install.iso.sha256 | grep -E '[0-9a-f]{5,40}' | head -n1) \
  DISTRI=opensuse \
  FLAVOR=offline-installer \
+ FULLURL=1 \
  ISO=Leap-16.0-offline-installer-ppc64le-Build1.3.install.iso \
+ MIRROR_HTTP=http://openqa.opensuse.org/assets/repo/Leap-16.0-aarch64-ppc64le-s390x-x86_64-Build1.3 \
+ MIRROR_HTTPS=https://openqa.opensuse.org/assets/repo/Leap-16.0-aarch64-ppc64le-s390x-x86_64-Build1.3 \
+ MIRROR_PREFIX=http://openqa.opensuse.org/assets/repo \
  REPO_0=Leap-16.0-aarch64-ppc64le-s390x-x86_64-Build1.3 \
- REPO_SLE_LEAP_16_0_AARCH64_PPC64LE_S390X_X86_64=Leap-16.0-aarch64-ppc64le-s390x-x86_64-Build1.3 \
+ REPO_LEAP_16_0_AARCH64_PPC64LE_S390X_X86_64=Leap-16.0-aarch64-ppc64le-s390x-x86_64-Build1.3 \
```

Comment:
In this case, we might want to have a different name for this repo (or have it duplicated to... i.e `LEAP_REPO`) would that be possible? See where its used at:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22054/files#diff-dd7d062b30fb82649a199111da8e89224600e38f86a24a23d251e0adc6747a89R22

Reply:
so it is basically exactly how it is written in xml, which is exactly how it is configured in OBS. Do we really need another mapping from there to some custom name? Maybe rename it in OBS if it is too ugly?

Reply:
Quoting from our conversation 
> we should aim to have REPO_OSS or maybe REPO_LEAP_OSS (because the folder is 000productcompose:leap_oss)